cookbook/common/parent.mk

```diff
@@ -1,7 +1,7 @@
 .SILENT:
 
 # This is a Makefile you can include in directories that have child directories you want to build common flyte targets on.
-SUBDIRS = $(shell ls -d */)
+SUBDIRS = $(shell find ./* -maxdepth 1 -type d)
```

Comment:
Instead of changing anything here, can we add a similar make file [like this](https://github.com/flyteorg/flytesnacks/blob/master/cookbook/integrations/Makefile) in each of those integrations?
This way we can always maintain `make all_register` at any "parent" level... and `make register` at a child level...